go/interpreter/lfvm/instructions.go

```diff
@@ -1083,10 +1083,6 @@ func neededMemorySize(c *context, offset, size *uint256.Int) (uint64, error) {
 }
 
 func genericCall(c *context, kind tosca.CallKind) {
-	warmAccess, coldCost, err := addressInAccessList(c)
```

Comment:
How does this work, are access list addresses pre-loaded or is this behaviour hidden inside accessAccount now?